src/main/rx/spektrum.c

```diff
@@ -35,6 +35,10 @@
 #include "fc/config.h"
 #include "fc/fc_dispatch.h"
 
+#if defined(USE_SPEKTRUM_REAL_RSSI) || defined(USE_SPEKTRUM_FAKE_RSSI)
```

Comment:
Don't use build flags around header inclusion.

Reply:
OK, fixed. This only an attempt to minimize and adapt dependencies to configurations. But it looks cleaner without.